docs/maintain-guides-democracy.md

```diff
@@ -188,27 +188,43 @@ account you want to clear of delegations. Click "Submit transaction" and confirm
 
 ![undelegate](assets/democracy/undelegate.png)
 
-## Proxies
+## Governance Proxies
 
-Proxies can be used to participate in governance on behalf of a stash account. A governance proxy
-can perform any transaction from Democracy, Council, Technical Committee, and Treasry modules.
-Unlike delegation, the proxy is meant to act as a longer-term account that makes all
-governance-related transactions for funds held in a different account. Delegation is a logical
+Governance Proxies can be used to participate in governance on behalf of a stash account. A
+governance proxy can perform any transaction from Democracy, Council, Technical Committee, and
+Treasury modules. Unlike delegation, the proxy is meant to act as a longer-term account that makes
+all governance-related transactions for funds held in a different account. Delegation is a logical
 action, taken when you trust another account's judgement, while proxying is more of a recommended
 security practice for keeping your funds safe and using an active account with low funds instead.
 
-Learn more about proxies on the [proxies](learn-proxies) page.
+Learn more about proxies and other types of proxies on the [proxies](learn-proxies) page.
 
-### Setting a proxy
+### Why use a proxy?
 
-Setting a proxy involves submitting a single transaction, the transaction type "addProxy" from the
+The idea of a governance proxy introduces a level of security. When engaging in transactions with
+Democracy, Council, Technical Committee, and Treasury modules, a governance proxy uses its low funds
+sent from its stash acccount. This avoids a stash or primary account with larger funds from
+participating in governance.
```

Comment:
I would not say the account does not participate, rather, that it does so safely. In other words, it does not perform on-chain actions, a safer proxy does it for that account.

Reply:
```suggestion
directly participating in governance.
```